webhooks/pod_webhook.go

```diff
@@ -39,11 +44,24 @@ const (
 	ProbeReadiness                                       = "/readyz"
 	ProbeLiveness                                        = "/healthz"
 	ProbeInitialDelay                                    = 5
+	kubeProxyDeploymentName                              = "kube-proxy"
+	kubeProxyServiceAccountName                          = "open-feature-operator-kube-proxy"
+	kubeProxyServiceName                                 = "kube-proxy-svc"
+)
+
+var (
+	currentNamespace      = "open-feature-operator-system"
+	kubeProxyImage        = "ghcr.io/open-feature/kube-flagd-proxy"
+	kubeProxyTag          = "v0.1.2"
```

Comment:
following https://github.com/open-feature/open-feature-operator/issues/415 this version should also be added to the renovate config